bpf/Makefile

```diff
@@ -194,7 +194,7 @@ bpf_lxc.o: bpf_lxc.ll
 
 subdirs: $(SUBDIRS)
 	$(QUIET) $(foreach TARGET,$(SUBDIRS), \
-		$(MAKE) $(SUBMAKEOPTS) -C $(TARGET))
+		$(MAKE) $(SUBMAKEOPTS) -C $(TARGET);)
```

Comment:
How do errors propagate out if this is separated by semicolons here? ie does the build fail if one subdirectory fails?

Reply:
Good question, I'll need to double check that.

Reply:
The build fails. This would normally not be the case, but we include the following from `../Makefile.defs`:
```
.SHELLFLAGS := -eu -o pipefail -c
```

I'll turn the `;` into a `&&` as an additional precaution. I also noticed that there are other `foreach` to address in a similar way, and will add them.